hasura/metadata/actions.yaml

```diff
@@ -1,4 +1,15 @@
 actions:
+- name: adminUpdateUser
+  definition:
+    kind: synchronous
+    handler: '{{HASURA_API_BASE_URL}}/actions/admin_update_user'
+    forward_client_headers: true
+    headers:
+    - name: verification_key
+      value_from_env: HASURA_EVENT_SECRET
```

Comment:
this isn't unique to this PR but is there some way we can do this automatically for all actions/events? also the env var has EVENT in the name which is a little misleading